test/test-uv-only.mjs

```
@@ -1,228 +1,244 @@
-    const pythonWorks = await testPython(pythonPath);
-    
-    if (pythonWorks) {
-      console.log('✓ Python execution successful');
-      testsPassed++;
-    } else {
-      console.error('✗ Python execution failed');
-      testsFailed++;
+  // -----------------------------------------------------------
+  // Test 7: Ensure ~/.platformio/penv has UV for other tests
+  // -----------------------------------------------------------
+  console.log('Test 7: Ensuring real penv has UV...');
```

ESLint

[error] Unexpected console statement.

(no-console)